developer/src/kmc-package/test/fixtures/invalid/warn_font_in_flo_does_not_have_default_ttf.kps

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package>
+  <System>
+    <KeymanDeveloperVersion>15.0.266.0</KeymanDeveloperVersion>
+    <FileVersion>18.0</FileVersion>
+  </System>
+  <Info>
+    <Name URL="">error_font_in_flo_does_not_have_default_ttf</Name>
+    <Copyright URL="">© 2019 National Research Council Canada</Copyright>
+    <Author URL="mailto:Eddie.Santos@nrc-cnrc.gc.ca">Eddie Antonio Santos</Author>
+    <Version>1.0</Version>
+  </Info>
+  <Files>
+    <File>
+      <Name>basic.kmx</Name>
+      <Description>Keyboard Basic</Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.kmx</FileType>
+    </File>
+    <!-- warn_font_in_flo_does_not_have_default_ttf: this uses a mismatching font, but
+         we won't get that far. This handles the scenario where FLO previously lists a font
+         but no ttf is available -->
```

Comment:
In the future when FLO makes updates, are there scenarios that could that impact some text fixture assumptions? 

For example, a listed font that previously doesn't have ttf changes to include one?

Reply:
The fixtures are all stored offline to avoid this arising (and have performant and consistent tests), see https://github.com/keymanapp/keyman/blob/c6baddad1237e09a6cdc14ccc64dd6d63253d63a/developer/src/kmc-package/test/fixtures/online/fonts.languagetechnology.org/%23families.json for example